frontend/apps/reader/modules/readerwikipedia.lua

```diff
@@ -5,7 +5,7 @@ local DEBUG = require("dbg")
 local _ = require("gettext")
 
 -- Wikipedia as a special dictionary
-local ReaderWikipedia = ReaderDictionary:extend{
+local ReaderWikipedia = ReaderDictionary:extend {
```

Comment:
We should keep the coding style consistent within the project :) If you take a look at other files, there is no space between a function call and `{`.